app/models/folio/request_abilities.rb

```diff
@@ -60,47 +60,39 @@ def pageable?
       !mediateable? && !hold_recallable? && any_holdings_pageable?
     end
 
-    def pickup_libraries
-      return (default_pickup_libraries + additional_pickup_libraries).uniq if location_restricted_service_point_codes.empty?
+    # FOLIO
+    def pickup_destinations
+      return (default_pickup_service_points + additional_pickup_service_points).uniq if location_restricted_service_point_codes.empty?
 
-      Settings.libraries.keys.select do |key|
-        location_restricted_service_point_codes.include? Settings.libraries[key].folio_pickup_service_point_code
-      end.map(&:to_s)
+      location_restricted_service_point_codes
     end
 
     # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
-    def default_pickup_library
-      service_points = Folio::Types.instance.service_points.select do |_k, v|
+    def default_pickup_destination
+      return 'EAST-ASIA' if request.origin_location&.match?(Regexp.union(/^EAL-SETS$/, /^EAL-STKS-/))
+
```

Comment:
```suggestion
```

This was removed in https://github.com/sul-dlss/sul-requests/pull/1648

Reply:
Noted!